src/components/budget/dto/budget-record.dto.ts

```diff
@@ -30,8 +32,10 @@ export class BudgetRecord extends IntersectionType(ChangesetAware, Resource) {
   @Field(() => Budget)
   declare readonly parent: BaseNode;
 
+  @Calculated()
```

Comment:
@CarsonF - Besides notifying the perm dumper that these are not applicable, what else does adding these `@Calculated` annotations do?

Reply:
Technically, edit/create/delete checks are forced to false, instead of checking policies.
https://github.com/SeedCompany/cord-api-v3/blob/5b1b85b3922230b3937509b98fde74eede1da735/src/components/authorization/policy/executor/policy-executor.ts#L36-L44

In all of these cases here, we are not checking the perms, or the properties we do check are not updatable.